mcs/class/corlib/Test/System/MathTest.cs

```diff
@@ -405,7 +405,7 @@ public void TestFloor ()
 		public void TestIEEERemainder ()
 		{
 			double a = Math.IEEERemainder (y, x);
-			double b = 0.0050000000000007816;
+			double b = 0.0050000000000010592;
 
```

Comment:
These values differ by one bit each in the exponent and mantissa:

```
0.0050000000000007816 -> 0x0018ae47e17a743f
0.0050000000000010592 -> 0x4019ae47e17a743f
```

I think the original value is correct because given the definitions of remainder and modulus:

```
remainder = x - (y * round (x / y))
modulus = (abs (x) - (abs (y) * (floor (abs (x) / abs (y))))) * sign (x)
```

In this case (x = 12.345, y = 0.1234) I think they should be the same. I don’t how rounding modes may affect this, though.